python/ray/serve/handle.py

```diff
@@ -181,6 +181,10 @@ def _options(
             multiplexed_model_id=multiplexed_model_id,
             stream=stream,
         )
+
+        if self._router is None:
```

Comment:
eager to create router to avoid race condition that concurrent multiplexed request trigger routers creation multiple times.

Reply:
pls remove from this PR as we discussed